python/cuml/model_selection/__init__.py

```diff
@@ -0,0 +1,8 @@
+from cuml.model_selection._split import _stratify_split
+from cuml.model_selection._split import _approximate_mode
+from cuml.model_selection._split import train_test_split
+
+
+__all__ = ['_stratify_split',
```

Comment:
Should `_stratify_split` and `_approximate_mode` be part of the publicly-declared API?

Reply:
from the naming of the functions, it doesn't seem like they should be - I'll change that